crates/uv-installer/src/installer.rs

```diff
@@ -36,6 +38,15 @@ impl<'a> Installer<'a> {
         }
     }
 
+    /// Set the `installer_name` to something other than `"uv"`.
+    #[must_use]
+    pub fn with_installer_name(self, installer_name: Option<&str>) -> Self {
```

Comment:
Do you mind if I change this to `Option<String>` to make it clear that this needs owned data and avoid the hidden clone?

Reply:
Don't mind it at all.